sdk/ed25519-instructions/src/lib.rs

```diff
@@ -190,15 +188,14 @@ fn get_data_slice<'a>(
 pub mod test {
     use {
         super::*,
-        crate::{
-            ed25519_instruction::new_ed25519_instruction,
-            hash::Hash,
-            signature::{Keypair, Signer},
-            transaction::Transaction,
-        },
         hex,
         rand0_7::{thread_rng, Rng},
         solana_feature_set::FeatureSet,
+        solana_hash::Hash,
+        solana_sdk::{
+            signer::{keypair::Keypair, Signer},
```

Comment:
Should this reference the individual `solana-signer` and `solana-keypair` crates?

Reply:
updated, this PR was opened before those crates landed